src/Sarif.Converters/FxCopConverter.cs

```diff
@@ -422,14 +477,26 @@ public void RefineMessage(string checkId, string typename, string messageId, str
             public void RefineIssue(string message, string result, string certainty, string level, string path, string file, int? line)
```

Comment:
>result [](start = 59, length = 6)

Can we rename local variable and parameter names `result` to `resolutionName` in this class, wherever it occurs? It took me reading down to here to understand that `result` wasn't a SARIF `Result` object.